go/pkg/client/tree.go

```diff
@@ -1,4 +1,3 @@
-// Package tree provides functionality for constructing a Merkle tree of uploadable inputs.
```

Comment:
Maybe just move this under "package" but leave the comment?

Reply:
This is a remnant of the time when tree used to actually be a separate package. Now that it's a part of client, I see no point to have a top-of-file comment, because that would just confuse the godoc (which comment should it show when generating the client docs?). I'm surprised this did not fail our external gofmt.

Reply:
I meant not as a top of file doc -- maybe something after imports. To still have some context, we already have too little documentation as is :)

Reply:
Okay, we'll see if that works.